contracts/src/bridge/L1/ObscuroBridge.sol

```diff
@@ -87,7 +87,7 @@ contract ObscuroBridge is
         address asset,
```

Comment:
unrelated. Let's rename this to `TenBridge`

Reply:
I'll rename in a separate PR as it will touch in a lot of places and obfuscate this one